tutorials/Additive_and_multiplicative_bias.ipynb

```diff
@@ -939,19 +943,19 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Percent bias\n",
+    "## Percent bias\n",
     "\n",
     "Let's imagine that fcst.temp_scrn represents the forecasted streamflow values and obs.temp_scrn represents the observed streamflow values. Now, we want to create another dataset with significantly higher values by adding 1000 to both the observed and forecasted streamflow values."
```

Comment:
```suggestion
    "Let's imagine that `fcst.temp_scrn` represents the forecasted streamflow values and `obs.temp_scrn` represents the observed streamflow values. Now, we want to create another dataset with significantly higher values by adding 1000 to both the observed and forecasted streamflow values."
```
@durgals - should fcst.temp_scrn and obs.temp_scrn be in backticks? I honestly don't know, I just wanted to ask.

Reply:
@Steph-Chong: Yes please.

Reply:
Thanks for letting me know - I have added the backticks.